variables.tf

```diff
@@ -106,6 +106,12 @@ variable "container_environment_variables" {
   default     = [""]
 }
 
+variable "use_canary" {
+  type = bool
+  description = "Uses canary deployment for Presto"
+  default = true
```

Comment:
*default* should be 'true' or 'false'?

Reply:
Before I added the switch it was enabled so I guess `true` makes sense. Or what do you think?

Reply:
I think false is good. canary requires double resources...